astropy/units/tests/test_structured.py

```diff
@@ -109,6 +109,21 @@ def test_extreme_recursive_initialization(self):
             'l',
         )  # fmt: skip
 
+        dt = np.dtype(
+            [("t", "f8"),
+             ("pvhd1d2",
+              ([("p", "f8"), ("v", "f8"), ("hd1d2",
+                                           [("h", "f8"), ("d1d2",
+                                                          [("d1", "f8"), ("d2", "f8")]),
+                                            ]),
```

Comment:
I see this isn't black formatted. That's fine, but I find the current format choice a bit opaque as well.

Reply:
I also didn't like it that much -- but with black the construct fills almost the whole screen and is even less readable (this is the standard "emacs formatting"). Anyway, leaving it for now...